conference/templatetags/conference.py

```diff
@@ -26,11 +26,13 @@ def _request_cache(request, key):
         request._conf_cache = {key: {}}
     return request._conf_cache[key]
 
-# Current conference object
-@register.assignment_tag()
```

Comment:
```
Deprecated since version 1.9: 
simple_tag can now store results in a template variable and should be used instead.
```
https://docs.djangoproject.com/en/1.11/howto/custom-template-tags/#assignment-tags